src/main/java/it/academy/app/validators/Available14To18Validator.java

```diff
@@ -0,0 +1,12 @@
+package it.academy.app.validators;
```

Comment:
```suggestion
@Size(min = 256, max = 1024)
@NotNull
@NotEmpty
private String answerAvailable14To18;
```
^^ pridejus i ApplicationForm klase gal nereiktu ir sito validatoriaus?

Reply:
Jau buvo sutvarkyta